slangpy/tests/utils/test_texture_loader.py

```
@@ -253,6 +253,47 @@ def test_load_texture_from_bitmap_file(device_type: spy.DeviceType, filename: st
     assert np.all(np.array(bitmap, copy=False) == np.array(bitmap_ref, copy=False))
 
 
+@pytest.mark.parametrize("device_type", helpers.DEFAULT_DEVICE_TYPES)
+def test_load_rgb_float_texture_with_generated_mips_extends_to_rgba(device_type: spy.DeviceType):
+    device = helpers.get_device(type=device_type)
+
+    format_support = device.get_format_support(Format.rgba32_float)
+    if (
+        not FormatSupport.texture in format_support
```

Ruff (0.15.12)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)